core/src/key/database/ac.rs

```diff
@@ -15,43 +15,43 @@ pub struct Tk<'a> {
 	_c: u8,
 	_d: u8,
 	_e: u8,
-	pub tk: &'a str,
+	pub ac: &'a str,
 }
 
-pub fn new<'a>(ns: &'a str, db: &'a str, tk: &'a str) -> Tk<'a> {
-	Tk::new(ns, db, tk)
+pub fn new<'a>(ns: &'a str, db: &'a str, ac: &'a str) -> Ac<'a> {
+	Ac::new(ns, db, ac)
 }
 
 pub fn prefix(ns: &str, db: &str) -> Vec<u8> {
 	let mut k = super::all::new(ns, db).encode().unwrap();
-	k.extend_from_slice(&[b'!', b't', b'k', 0x00]);
+	k.extend_from_slice(&[b'!', b'a', b'c', 0x00]);
 	k
 }
 
 pub fn suffix(ns: &str, db: &str) -> Vec<u8> {
 	let mut k = super::all::new(ns, db).encode().unwrap();
-	k.extend_from_slice(&[b'!', b't', b'k', 0xff]);
+	k.extend_from_slice(&[b'!', b'a', b'c', 0xff]);
 	k
 }
 
-impl KeyRequirements for Tk<'_> {
+impl KeyRequirements for Ac<'_> {
 	fn key_category(&self) -> KeyCategory {
-		KeyCategory::DatabaseToken
+		KeyCategory::DatabaseAccess
 	}
 }
 
-impl<'a> Tk<'a> {
-	pub fn new(ns: &'a str, db: &'a str, tk: &'a str) -> Self {
+impl<'a> Ac<'a> {
+	pub fn new(ns: &'a str, db: &'a str, ac: &'a str) -> Self {
 		Self {
 			__: b'/',
 			_a: b'*',
 			ns,
 			_b: b'*',
 			db,
 			_c: b'!',
-			_d: b't',
-			_e: b'k',
-			tk,
+			_d: b'a',
```

Comment:
This would require a migration

Reply:
That is correct. Is that not acceptable for a breaking change in 2.0?

Reply:
Marking as resolved as per other comment